calling/build.gradle.kts

```diff
@@ -0,0 +1,49 @@
+plugins {
+    Plugins.androidLibrary(this)
+    Plugins.multiplatform(this)
+}
+
+group = "com.wire.kalium"
+version = "0.0.1-SNAPSHOT"
+
+android {
+    compileSdk = Android.Sdk.compile
+    sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
+    defaultConfig {
+        minSdk = Android.Sdk.min
+        targetSdk = Android.Sdk.target
+        consumerProguardFiles("consumer-proguard-rules.pro")
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+}
+
+kotlin {
+    jvm {
+        compilations.all {
+            kotlinOptions.jvmTarget = "1.8"
+            kotlinOptions.freeCompilerArgs += "-Xopt-in=kotlin.RequiresOptIn"
+        }
+        testRuns["test"].executionTask.configure {
+            useJUnit()
+        }
+    }
+    android()
+
+    sourceSets {
+        val androidMain by getting {
+            dependencies {
+                implementation(Dependencies.Calling.avs)
+                implementation(Dependencies.Calling.jna)
+                implementation(Dependencies.Cryptography.cryptoboxAndroid)
```

Comment:
Why do we have a dependency on `cryptoboxAndroid`?

Reply:
@typfel  libsodium is needed to use AVS

Reply:
Libsodium is statically linked into the AVS library there no need to provide it at runtime.